src/Microsoft.Crank.Controller/Program.cs

```diff
@@ -3351,7 +3388,11 @@ private static async Task<string> GetRelayTokenAsync(Uri endpointUri)
             }
 
             var rcsb = new RelayConnectionStringBuilder(connectionString);
-            var tokenProvider = TokenProvider.CreateSharedAccessSignatureTokenProvider(rcsb.SharedAccessKeyName, rcsb.SharedAccessKey);
+            TokenProvider tokenProvider = null;
+            AccessToken token = default;
```

Comment:
I forgot to remove this one I removed the cert-based auth support on the controller side for Azure Relay. I did this because conditional access made it where you could only use cert-based auth on computers that could already access the resources that you cared about.